src/sdam/srv_polling.ts

```diff
@@ -67,8 +67,7 @@ export class SrvPoller extends TypedEventEmitter<SrvPollerEvents> {
     super();
 
     if (!options || !options.srvHost) {
-      // TODO(NODE-3483)
-      throw new MongoDriverError('Options for SrvPoller must exist and include srvHost');
+      throw new MongoInvalidArgumentError('Options for SrvPoller must exist and include srvHost');
```

Comment:
I think this one might be runtime, since the user can't directly instantiate SrvPoller

Reply:
For this one I called it a MongoInvalidArgumentError specifically because it gets instantiated in the constructor for `Topology` (src/sdam/topology.ts) which takes the options argument directly from the  `connect` function (src/operations/connect.ts:65).

I may be misunderstanding that `connect` function though, so totally open to fix that.

Reply:
Yeah it's a bit convoluted, but that connect function is actually invoked through the MongoClient constructor (it's imported in the mongo client file and then used by its public connect method), however the options passed to it are the ones parsed in when calling the MongoClient constructor, and that's where the user-facing validation error resides (see L71 in connection_string).. so, by the time we get to this code, it's our fault for not catching it because the user shouldn't really have to worry about what an SrvPoller is